bots/notify/src/main/java/org/openjdk/skara/bots/notify/issue/IssueNotifier.java

```diff
@@ -428,6 +405,23 @@ public void onNewOpenJDKTagCommits(HostedRepository repository, Repository local
         }
     }
 
+    private String getRequestedVersion(Repository localRepository, Commit commit, String tagBranch) {
```

Comment:
Minor: would `branch` be a better name? Or is it always a tag name in this method?

Reply:
I agree. It seems I was lazy and just went with the IDE generated extracted method signature.